lib/ddtrace/configuration/settings.rb

```diff
@@ -21,6 +22,24 @@ class Settings
               default: -> { env_to_bool(Ext::Runtime::Metrics::ENV_ENABLED, false) },
               lazy: true
 
+      # Look for all headers by default
+      option  :propagation_extract_style,
```

Comment:
This file is for global settings. is propagation style a global setting? Is it possible that it propagates traces between multiple services that each have their own style? (i.e. we need to support both styles simultaneously?)

Reply:
Yeah, this is meant to be a global setting. Most likely people won't adjust these settings at all, if they change anything it'll be the inject styles to add an additional style. This will likely be the case when they are using `dd-trace-rb` along side another library that only supports B3 headers so they get interoperability.

Reply:
Seems fine to me.